scripts/test/Muon/elemental_analysis/elemental_analysis_test.py

```diff
@@ -472,13 +462,9 @@ def test_checked_data_changes_all_states_in_list(self, mock_update_peak_data):
         self.assertTrue(all(map(lambda m: m.setChecked.call_count == 1, selection)))
         mock_update_peak_data.assert_called_with('Cu')
 
-    def test_get_electron_peaks_returns_a_dict_with_correct_length(self):
-        # TODO: modify this after merging #26444
-        peaks = len(self.gui.ptable.peak_data["Electrons"])
-        electron_dict = self.gui._get_electron_peaks()
-        self.assertEqual(len(electron_dict), peaks)
-        for _, peak in electron_dict.items():
-            float(peak)
+    def test_that_ptable_contains_no_peak_not_part_of_an_element(self):
+        self.assertTrue('Electrons' not in self.gui.ptable.peak_data)
+        self.assertTrue('Gammas' not in self.gui.ptable.peak_data)
```

Comment:
shall we test that electrons and gammas are stored in the elements?

Reply:
@LolloB can you make an issue for what we discussed about the data tests and reference it here? I will then approve this

Reply:
Created #26593